cloud-agent/service/server/src/main/scala/org/hyperledger/identus/didcomm/controller/DIDCommControllerError.scala

```diff
@@ -3,49 +3,39 @@ package org.hyperledger.identus.didcomm.controller
 import org.hyperledger.identus.mercury.model.DidId
 import org.hyperledger.identus.shared.models.{Failure, StatusCode}
 
-sealed trait DIDCommControllerError(
-    val statusCode: StatusCode,
-    val userFacingMessage: String
-) extends Failure {
-  override val namespace = "DIDCommControllerError"
+sealed trait DIDCommControllerError extends Failure {
+  override def namespace = "DIDCommControllerError"
```

Comment:
Any particular reason why you changed the previous construct and the below case class declaration? If not, I propose to stay consistent with the rest of the codebase.

Reply:
To make `final case class RecipientNotFoundError()` an object

Reply:
Let's keep it as a case class, then. I don't think it is worth the trouble. Who knows, maybe we will need to some kind of message ID as a parameter in the future.